src/test/java/macrobase/pipeline/BasicBatchedPipelineTest.java

```diff
@@ -18,8 +16,8 @@
 
 import static org.junit.Assert.*;
 
-public class BatchAnalyzerTest {
-    private static final Logger log = LoggerFactory.getLogger(BatchAnalyzerTest.class);
+public class BasicBatchedPipelineTest {
```

Comment:
Could be useful to have a short comment describing what this test is trying to test. (Relevant for most testing files, but especially the end-to-end ones)